docs/concepts/controllers/statefulsets.md

```diff
@@ -0,0 +1,163 @@
+---
```

Comment:
Thanks for the list. We should have the same list of assignees in all StatefulSet docs